### ISSUE TITLE
Nerf Dexterity Leveling

### ISSUE THREAD

**Issue by HiTher3** [Tslat/Advent-Of-Ascension#3569]
The Dexterity Skill, as seen in the skills menu when you press the Delete key.
It can give various mobility-based attributes and abilities when leveled up.

The Dexterity Skill levels up **way too fast,** to the point where just casually building a house can get you up to level 15, while all the other skills still sit in the 1-3 range.
I've noticed that when you gain experience for Dexterity it's a lot more experience than most other skills; for example, mining a log gives you 0.11 Extraction experience, meanwhile Dexterity gives you over 5 Dexterity experience every time you gain some, meaning you'd essentially have to break 50 logs to match that level of experience.
I think this could be fixed by simply lowering the amount of experience that you get for Dexterity; I'd personally suggest around 0.5 to 1.5 Dexterity experience since while it would still likely be the fastest leveling skill, it wouldn't be nearly as overpowered.

A slightly different way that I think could work is maybe determining the amount of Dexterity experience you gain by how much you moved around in that time. So if you're running around for a while you'd gain a lot of Dexterity experience, but if you were just sitting around in your base, building something, or generally not moving a lot, you'd gain very little Dexterity experience.

I imagine that this probably has been talked about before, but I just feel that they don't really address the problem. Most of the time, it's just a passing mention, and the few times people do criticize it, it's often buried beneath hundreds or thousands of other comments. I just felt that I needed to mention this because I think that, while it isn't that big of an issue, is still something that should be fixed regardless.

*I always dread doing school essays yet I typed this whole thing in 20 minutes like it was nothing. I wish I had that kind of motivation all the time lol*

**Comment by Cruuk** [comment]
Are you on 1.16.5? It was already nerfed somewhere in 1.19. To give you an idea, sprint jumping at level 1 gives around ~6 xp every 10 seconds in 1.16, and sprint jumping at level 1 gives around ~3 xp every 10 seconds in 1.19+.

It's still fast, but that's the point, it's a passive skill that, along with Innervation, is supposed to level up as you progress through the mod itself, it just so happens to tie to a very fundamental aspect of MC: running. Innervation is slightly slower, but it can easily pick up and contend with Dex's speed when you get outside of OW and start killing mobs with higher HP/fight bosses.

It being fast allows players to get its benefits faster, which allows content to be designed more easily assuming players will have more substantiated Dex/Innervation levels, in other words more fun content designed around players being faster/having more HP.

Let me repeat myself, the idea of Dex/Innervation being fast is that stuff like lategame content will be balanced around players having having something like 30-50+ levels of Dex/Innervation, you never need to go out of the way for the skills unless you use specific grinding methods.
And even if you do grind Dex to like 40 before leaving OW, another key design element is that Dex's abilities increment in potency *very* slowly. Even if there is a difference, level 70 dex players aren't gonna have a super massive advantage compared to level 30 dex players
The only ability that isn't just a number slowly rising is the double jump, which is supposed to be a big reward for training that far. I do not know how high you got your Dex level, but you can't judge the exp curve from an earlygame standpoint which is supposed to be quick. Even pre-nerf, it starts to slow down around 50, and really slows down at 80. It allows you to get the fundamentals somewhat quickly, but makes the rest of it a challenge.

You have to make it possible to get to 100 in a tolerable amount of time. Compare it to its old counterpart, which took an hour to get from level 1-3. Because of the curve, effectively **no one** got it past 80+. The new skills are made so it's not super hard to get to 100, so you can actually feasibly get all the abilities in a survival world, because they're supposed to be fun, not an MMORPG grind. Besides, there's still level 1000/Cycling for the people that want to emulate that grind. Imagine trying to get to 1000 with a far slower level curve LOL.

You can't really compare it to Extraction, which is an optional active skill you have to go out of your way to train, but gives more substantiated rewards. Also, a log giving 0.11 xp is because you've barely scratched the surface level of how you can increase that.

There's: the thousands of blocks you break per world in general, harder blocks like Obsidian giving way more xp, faster picks, the efficiency enchantment, haste, Pickmax, Occult Axe, Energistic's boost, the Harvest Speed boost ability, Smelting in general giving good amounts of xp, and more. You can easily get Extraction higher than Dex, and players who put the work in and are familiar with Extraction's training mechanics should be rewarded more than someone who just breaks blocks with an unenchanted tool without any boosts.

In other words, Dex being fast just allows things to be more fun, even if that makes it and Innervation reach higher levels on average than the other more active skills
